WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -845,12 +952,17 @@ - (void)configureCell:(UITableViewCell *)aCell atIndexPath:(NSIndexPath *)indexP
 
 - (CGFloat)tableView:(UITableView *)tableView estimatedHeightForRowAtIndexPath:(NSIndexPath *)indexPath
 {
+    // Note:
+    // Due to a UITableView bug on iOS 8, let's keep the estimated height to the bare minimum.
+    // If the estimated is bigger than needed, UITableView might actually use that size,
+    // and shrink using an undesired animation.
+    
```

Comment:
Honestly, i've initially added this in the spot where the const is defined. But it looked awful. However, you're actually right, maybe this might end up unread. Updating, thanks!